### ISSUE TITLE
How do I know the target category that corresponds to the tag value in labels.txt file?

### ISSUE THREAD

**Issue by yjl326** [vvgoder/SEU_PML_Dataset#1]
Hi authors, I downloaded your dataset and read your paper. It says that the target category of the dataset is 13, but I didn't find any labels_category value greater than 3 in the labels.txt file of your dataset. I'm really confused and don't know what the target category is. So could you please provide the labels_list for the labels values corresponding to the target category? Thanks!

**Comment by vvgoder** [comment]
Thank you for your attention to our work, and thank you for your reminding.  Recently, we will open source the sub category annotation, which will be updated in this repo link!

**Comment by vvgoder** [comment]
> Hi authors, I downloaded your dataset and read your paper. It says that the target category of the dataset is 13, but I didn't find any labels_category value greater than 3 in the labels.txt file of your dataset. I'm really confused and don't know what the target category is. So could you please provide the labels_list for the labels values corresponding to the target category? Thanks!

Thank you for your attention to our work, and thank you for your reminding. Recently, we will open source the sub category annotation, which will be updated in this repo link!

**Comment by wjq777777** [comment]
Hello author, I have downloaded your paper and data set, but I can't find the description of the type of image annotation in the data set, and I don't know what the specific type of target is. Could you please provide the label.txt file with the label description of the target type? Thank you very much!

**Comment by vvgoder** [comment]
Thank you for your interest in our research. In our annotation files, the bounding box (Bbox) and category information are stored in YOLO format. The category IDs 0-3 correspond to person, motor vehicle, non-motor vehicle, and privacy categories, respectively.

> Hello author, I have downloaded your paper and data set, but I can't find the description of the type of image annotation in the data set, and I don't know what the specific type of target is. Could you please provide the label.txt file with the label description of the target type? Thank you very much!

**Comment by wjq777777** [comment]
Thank you very much for your reply. In your paper, I found that you divided the categories into a total of thirteen categories, but in the dataset I downloaded from GitHub, the label file only has 4 classes. And the link on GitHub provided the train set and test set, but no val set. Is there no val set separated?

**Comment by vvgoder** [comment]
> Thank you very much for your reply. In your paper, I found that you divided the categories into a total of thirteen categories, but in the dataset I downloaded from GitHub, the label file only has 4 classes. And the link on GitHub provided the train set and test set, but no val set. Is there no val set separated?

Yes, I've been busy with deadlines recently, sorry for the late reply. I will provide the basis for splitting the train and validation datasets. Additionally, I will  upload the labels for the 13 classes in 1-2 weeks.

**Comment by wjq777777** [comment]
> > Thank you very much for your reply. In your paper, I found that you divided the categories into a total of thirteen categories, but in the dataset I downloaded from GitHub, the label file only has 4 classes. And the link on GitHub provided the train set and test set, but no val set. Is there no val set separated?
> 
> Yes, I've been busy with deadlines recently, sorry for the late reply. I will provide the basis for splitting the train and validation datasets. Additionally, I will upload the labels for the 13 classes in 1-2 weeks.

Hello, sorry to bother you again! I would like to ask when the data set of 13 types of labels will be updated? This data set is fantastic. We are very interested in this dataset and can't wait to use it.

**Comment by vvgoder** [comment]
> > > Thank you very much for your reply. In your paper, I found that you divided the categories into a total of thirteen categories, but in the dataset I downloaded from GitHub, the label file only has 4 classes. And the link on GitHub provided the train set and test set, but no val set. Is there no val set separated?
> > 
> > 
> > Yes, I've been busy with deadlines recently, sorry for the late reply. I will provide the basis for splitting the train and validation datasets. Additionally, I will upload the labels for the 13 classes in 1-2 weeks.
> 
> Hello, sorry to bother you again! I would like to ask when the data set of 13 types of labels will be updated? This data set is fantastic. We are very interested in this dataset and can't wait to use it.

Thank you for your reminder. I have uploaded the sub-category labels so far. It should be noted that we found some ambiguities in the sub-category labels of the dataset when using it later. We originally planned to carefully verify and make corrections. However, due to time constraints (such as completing a Ph.D.), we did not have time to make updates. If you find any errors, please contact us promptly, and we will make the necessary corrections.

**Comment by wjq777777** [comment]
> > > > Thank you very much for your reply. In your paper, I found that you divided the categories into a total of thirteen categories, but in the dataset I downloaded from GitHub, the label file only has 4 classes. And the link on GitHub provided the train set and test set, but no val set. Is there no val set separated?
> > > 
> > > 
> > > Yes, I've been busy with deadlines recently, sorry for the late reply. I will provide the basis for splitting the train and validation datasets. Additionally, I will upload the labels for the 13 classes in 1-2 weeks.
> > 
> > 
> > Hello, sorry to bother you again! I would like to ask when the data set of 13 types of labels will be updated? This data set is fantastic. We are very interested in this dataset and can't wait to use it.
> 
> Thank you for your reminder. I have uploaded the sub-category labels so far. It should be noted that we found some ambiguities in the sub-category labels of the dataset when using it later. We originally planned to carefully verify and make corrections. However, due to time constraints (such as completing a Ph.D.), we did not have time to make updates. If you find any errors, please contact us promptly, and we will make the necessary corrections.

Thank you so much for your update! Thank you for your work.
